display/background.py

```diff
@@ -19,28 +19,27 @@ def __init__(self, screen):
 
     def render(self):
 
-        self.screen.fill(BACKGROUND)
-
-        zucc = pygame.image.load(os.path.join("assets", "backgrounds", "bad", "ground.png"))
-
-        self.screen.blit(zucc, (0, screen_height - GROUND_HEIGHT))
-
         self.sprites.update(self.speed)
 
         if len(self.sprites) < self.building_amount:
             q = -99999
             for buildong in self.sprites:
                 if buildong.rect.x > q:
                     q = buildong.rect.x
-            self.add_building(random.randint(100, 800), q + self.building_width, width=self.building_width)
+            self.add_building(q + self.building_width, width=self.building_width)
+
+        self.screen.fill(BACKGROUND)
 
         self.sprites.draw(self.screen)
 
+        zucc = pygame.image.load(os.path.join("assets", "backgrounds", "city", "ground.png"))
```

Comment:
[INFO] Reference to zucc added. This is correct behaviour and will receive **praise**.